tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "isolatedModules": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "lib": [ "esnext" ]
+    "lib": [ "esnext", "DOM" ]
```

Comment:
@fredwes Where does this change come from? Node.js does not really expose the DOM libraries, so I'm wondering why this is required. 🤔

Reply:
@arthurschreiber this is because the Azure libs for the Azure key vault reference some types from DOM, TS will complain that it can't find those standard types without it

e.g.: the "Headers" type here: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-http/lib/fetchHttpClient.ts#L198